spec/jobs/socure_shadow_mode_proofing_job_spec.rb

```diff
@@ -373,6 +387,27 @@
         expect { perform }.to raise_error(JSON::ParserError)
       end
     end
+
+    context 'when an unknown reason code is encountered' do
+      let(:socure_response_body) do
+        super().tap do |body|
+          body[:kyc][:reasonCodes] << 'I000'
+        end
+      end
+
+      it 'still logs it' do
+        perform
+        expect(analytics).to have_logged_event(
+          :idv_socure_shadow_mode_proofing_result,
+          satisfy do |attributes|
+            errors = attributes.dig(:socure_result, :errors)
+            expect(errors).to include(
+              'I000' => '',
```

Comment:
blank strings are hard to see in cloudwatch I think, would it make sense to have some sort of `"[unknown]"` or other sentinel value?